fapi-client/src/main/scala/com/gu/facia/api/FAPI.scala

```diff
@@ -87,6 +87,19 @@ object FAPI {
     for(snapContent <- ContentApi.latestContentFromLatestSnaps(capiClient, allSnaps, adjustItemQuery))
       yield snapContent}
 
+  def getTreatsForCollection(collection: Collection, adjustSearchQuery: AdjustSearchQuery = identity, adjustItemQuery: AdjustItemQuery = identity)
+                      (implicit capiClient: GuardianContentClient, ec: ExecutionContext) = {
+    val treatsRequest = Collection.treatsRequestFor(collection)
+    ContentApi.buildHydrateQueries(capiClient, treatsRequest.treatIds, adjustSearchQuery) match {
+      case Success(hydrateQueries) =>
```

Comment:
I see what you mean, but this is the way `buildHydrateQueries` works, and this PR is not about refactoring that. If this format isn't good enough for `getTreatsForCollection`, when why is it good enough for `getContentForCollection`?
Perhaps the solution is to change the signature to `buildHydrateQueries` to return a `Response[Seq[SearchQuery]]` instead of a `Try[Seq[SearchQuery]]`.